ts/screens/wallet/payment/ErrorTransactionScreen.tsx

```diff
@@ -0,0 +1,189 @@
+/**
```

Comment:
I think you can extract renderErrorMessageType and renderErrorIcon outisde the component (but still in this file) because they are pure functions.

Reply:
I solved in the commit 092ff8f